### ISSUE TITLE
Wrong size in GenerationSession.log_probs

### ISSUE THREAD

**Issue by shenjiangqiu** [NVIDIA/TensorRT-LLM#490]
https://github.com/NVIDIA/TensorRT-LLM/blob/a21e2f85178111fed9812bb88c2cc7411b25f0ba/tensorrt_llm/runtime/generation.py#L713

currently, the size of the log_probs is (self.max_new_tokens, batch_size, scfg.num_beams), but it start writing at `context_length`, so the size is too small. it should be at least context_length+max_new_tokens.

**Comment by byshiue** [comment]
The logits of step "context_length + 1" is written in `self.logit[1]`. So, there is no issue. 

I will close this issue first. If you encounter problem, please provide the reproduced steps and reopen this issue.
